arches/install/requirements.txt

```diff
@@ -7,7 +7,7 @@ django-guardian==2.4.0
 python-memcached==1.59
 celery==5.3.6
 django-celery-results==2.5.1
-mapbox-vector-tile==1.2.0
+mapbox-vector-tile==2.0.1
```

Comment:
I wonder if we need this package at all anymore.

Reply:
Oh good call. Looks like TileStache was removed in arches 5. Maybe it was here for that?

Reply:
Yeah - that must be the case. Arches used to ship with a tile server using TileStache, but now we just let people implement their own and proxy tile requests through Arches.